app/main/views/index.py

```diff
@@ -80,6 +80,18 @@ def robots():
     ), 200, {'Content-Type': 'text/plain'}
 
 
+@main.route('/.well-known/security.txt')
+def security_txt():
+    return (
+        'Contact: mailto:security@cds-snc.ca\n'
+        'Contact: mailto:security@cds-snc.ca\n'
```

Comment:
```suggestion
        'Contact: mailto:securite@cds-snc.ca\n'
```
The email address is different in French

Reply:
Good catch, I'll fix that now 👍

Reply:
Fix should be in place now